connectivity/service/src/main/java/org/eclipse/ditto/connectivity/service/messaging/httppush/AsyncJwtLoader.java

```diff
@@ -147,21 +148,34 @@ private DittoRuntimeExceptionBuilder<JwtInvalidException> getJwtInvalidException
     private static HttpRequest toTokenRequest(final String tokenEndpoint,
             final String clientId,
             final String clientSecret,
-            final String scope) {
+            final String scope,
+            final Optional<String> audience) {
 
-        final var body = asFormEncoded(clientId, clientSecret, scope);
+        final var body = asFormEncoded(clientId, clientSecret, scope, audience);
         final var entity = HttpEntities.create(ContentTypes.APPLICATION_X_WWW_FORM_URLENCODED, body);
         return HttpRequest.POST(tokenEndpoint).withEntity(entity).addHeader(ACCEPT_JSON);
     }
 
-    private static String asFormEncoded(final String clientId, final String clientSecret, final String scope) {
+    private static String asFormEncoded(final String clientId, final String clientSecret, final String scope, final Optional<String> audience) {
```

Comment:
here as well, please don't use `Optional` as argument

Reply:
updated it